### DISCUSSION TITLE
Tried everything however I cannot install AIO Patch

### DISCUSSION THREAD

**Discussion by romz11** [Mr-MIBonk/M.I.B._More-Incredible-Bash#123] [Q&A]
My firmware is:

MHI2Q_US_AUG22_P5087
MU version: 1316

I uploaded all of the patch files to my SD card since it would automatically select the right one however when running the AIO patch, the following pops up: 
![12](https://user-images.githubusercontent.com/126043455/220518776-1f9efca4-4ebf-48fd-9ef7-aa91789da52d.jpg)
![13](https://user-images.githubusercontent.com/126043455/220518782-76c5e464-cbcd-4fc0-a4bb-3a6c58bfec8d.jpg)
![patch](https://user-images.githubusercontent.com/126043455/220518848-eb381092-d29f-4903-8f68-3adf67a2ece6.png)

No idea what's wrong since I do have the correct patch inside my SD card inside the Patch folder.

**Comment by harman-f** [comment]
Well, as written on screen:
![image](https://user-images.githubusercontent.com/98130152/221414472-5398ff9d-65ef-4944-9106-bf0a0110fdb8.png)
Your ifs-root is already patched.
